app/serializers/v1/workflow/process_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class ProcessSerializer
```

Comment:
if you look at all the other serializers, they inherit from `< ApplicationSerializer`.
That's going to put some conventions on your models that will make them error out.  Basically any model that goes into the API needs to have a non-incremental integer ID called `external_identifier`.  That's going to be a bit annoying though bc now we need to update the migrations and ` include ApplicationRecord::ExternalIdentifier` in the models.  See this model for example: https://github.com/keithtom/wildflower-platform/blob/main/app/models/address.rb

Also check out this migration I wrote for a similar prob:
https://github.com/keithtom/wildflower-platform/blob/main/db/migrate/20211117190245_add_external_identifiers.rb

don't need to include JSONAPI:Serializer either.

https://github.com/keithtom/wildflower-platform/blob/main/app/serializers/application_serializer.rb